src/components/Modal/SettingsModal.tsx

```diff
@@ -182,28 +187,41 @@ const DeveloperButton = (props: { t: i18nTranslate }) => {
           ? isTestnets
             ? t?.('enableMainnets') || 'Enable mainnets'
             : t?.('enableTestnets') || 'Enable testnets'
-          : `(${5 - count})`
+          : `(${t?.('clickMoreTimes', { n: (5 - count).toString() }) || 5 - count})`
       }
     />
   )
 }
 
-const ToolsButton = (props: { t: i18nTranslate }) => {
-  const { t } = props
+const CustomButton = (props: { type?: CustomButtonType, t: i18nTranslate }) => {
+  const { type: buttonType, t } = props
+
+  // Default Custom Button (Tools):
+  let link = 'https://tools.pooltogether.com'
```

Comment:
Rather than having the package consumer pass `core` why not just pass this "link data"?
```ts
interface LinkData {
  link: stirng
  title: string
  description: string
}
```
(or name the paramters whatever the parameters are name on the buttons themselves so it matches)

Reply:
Implemented in 0799aff!

This way it also means less editing of this package, which is nice.